src/wallet/rpcwallet.cpp

```diff
@@ -4089,7 +4089,7 @@ static RPCHelpMan send()
                 UniValueType(), // outputs (ARR or OBJ, checked later)
                 UniValue::VNUM, // conf_target
                 UniValue::VSTR, // estimate_mode
-                UniValue::VNUM, // fee_rate
```

Comment:
How about removing this `RPCTypeCheck`?

Reply:
I followed the practice seen in the file and the codebase when grepping `UniValueType()` and it seems to have documentation value for future readers of the code.

Reply:
That's fine. I just want to point that if removing some code makes no difference then it's because its redundant or some test is missing.